src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -136,6 +136,8 @@
 
 #ifdef NGRAPH_DISTRIBUTED_ENABLE
 #include "ngraph/runtime/reference/allreduce.hpp"
+#include "ngraph/runtime/reference/broadcastrecv.hpp"
```

Comment:
rename files to `broadcast_xxx`
Can't we spell out `receive`?
If you want the names to be the same length then `broadcast_tx` and `broadcast_rx` always works

Reply:
Just followed the naming convention of _send/_recv for interconnect communication.